src/warnet/tank.py

```diff
@@ -116,30 +124,37 @@ def ipv4(self):
             self._ipv4 = generate_ipv4_addr(self.warnet.subnet)
         return self._ipv4
 
+    @property
+    def autonomous_system(self):
+        if self._a_system is None:
+            self._a_system = generate_as(self.warnet)
+        return self._a_system
+
     @property
     def container_name(self):
         if self._container_name is None:
             self._container_name = (
-                f"{self.docker_network}_{CONTAINER_PREFIX_BITCOIND}_{self.suffix}"
+                f"{self.network_name}_{CONTAINER_PREFIX_BITCOIND}_{self.suffix}"
             )
         return self._container_name
 
     @property
     def exporter_name(self):
         if self._exporter_name is None:
             self._exporter_name = (
-                f"{self.docker_network}_{CONTAINER_PREFIX_PROMETHEUS}_{self.suffix}"
+                f"{self.network_name}_{CONTAINER_PREFIX_PROMETHEUS}_{self.suffix}"
             )
         return self._exporter_name
 
     @property
     def container(self) -> Container:
+        # cast makes typecheckers happy
         if self._container is None:
             try:
-                self._container = docker.from_env().containers.get(self.container_name)
+                self._container = cast(Container, docker.from_env().containers.get(self.container_name))
```

Comment:
I was grepping for leftover `docker.from_env()` calls and I just wanna make sure that this line here is still abstractable

Reply:
nice catch, looks like this was fixed